opal/mca/pmix/cray/pmix_cray.c

```diff
@@ -253,6 +253,7 @@ static void cray_get_more_info(void)
     opal_setenv("OMPI_NUM_APP_CTX", pbuf, true, &environ);
     opal_setenv("OMPI_FIRST_RANKS", firstrankstring, true, &environ);
     opal_setenv("OMPI_APP_CTX_NUM_PROCS", npstring, true, &environ);
+    opal_setenv(OPAL_MCA_PREFIX "orte_bound_at_launch", "true", true, &environ);
```

Comment:
On master, would you please move this to the orte/mca/schizo/alps component? I know it has to go here for v2.1, but it really doesn't belong here. Thanks!